filters/flowid/filter.go

```diff
@@ -54,16 +72,13 @@ func (spec *flowIdSpec) CreateFilter(fc []interface{}) (filters.Filter, error) {
 		} else {
 			return nil, filters.ErrInvalidFilterParameters
 		}
-	}
-	var flowIdLength = defaultLen
-	if len(fc) > 1 {
-		if l, ok := fc[1].(float64); ok && l >= MinLength && l <= MaxLength {
-			flowIdLength = int(l)
-		} else {
-			return nil, filters.ErrInvalidFilterParameters
+		if len(fc) > 1 {
+			log.Println("flow id filter warning: this syntaxt is deprecated and will be removed soon. " +
+				"please check updated docs")
 		}
 	}
-	return &flowId{reuseExisting, flowIdLength}, nil
+	return &flowId{reuseExisting: reuseExisting, generator: spec.generator}, nil
 }
 
-func (spec *flowIdSpec) Name() string { return Name }
+// Name returns the canonical filter name
+func (_ *flowIdSpec) Name() string { return Name }
```

Comment:
i see that in the same cloud of services one would want just a single type of flowId, but for the record: i would have allowed the custom implementations to have different names. Since they can still do it with completely independent filters, it's cool maybe even better so.